### PR TITLE
Andres markets page

### PR DESCRIPTION
**Pull request by AndresImach**

### CODE REVIEW

File: src/components/Banner/OptionalLinkBanner/OptionalLinkBanner.tsx
Comment (TomasCImach) [inline src/components/Banner/OptionalLinkBanner/OptionalLinkBanner.tsx:1]: rename file to index.tsx to comply with naming standards.

https://discord.com/channels/798007956239024148/1288946967728357501/1289250156482400267

File: src/components/Banner/PromotedBanner/PromotedBanner.tsx
Comment (TomasCImach) [inline src/components/Banner/PromotedBanner/PromotedBanner.tsx:1]: rename file to index.tsx to comply with naming standards.

https://discord.com/channels/798007956239024148/1288946967728357501/1289250156482400267

File: src/components/MarketCards/MarketCards.tsx
Comment (TomasCImach) [inline src/components/MarketCards/MarketCards.tsx:1]: rename file to index.tsx to comply with naming standards.

https://discord.com/channels/798007956239024148/1288946967728357501/1289250156482400267

File: package.json [hunk 1]
```diff
@@ -10,10 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-select": "^2.1.2",
     "@radix-ui/react-dialog": "^1.1.1",
     "@radix-ui/react-slider": "^1.2.0",
     "@radix-ui/themes": "^3.1.3",
     "@reduxjs/toolkit": "^2.2.7",
+    "lucide-react": "^0.451.0",
```
Comment (TomasCImach) [inline package.json:19]: this dependency is not being used

File: package.json [hunk 2]
```diff
@@ -23,9 +26,12 @@
     "react-feather": "^2.0.10",
     "react-redux": "^9.1.2",
     "react-router-dom": "^6.26.1",
+    "react-swiper": "^0.1.5",
```
Comment (TomasCImach) [inline package.json:29]: this dependency is not being used

**Review by TomasCImach — CHANGES_REQUESTED** [review]

**Review by TomasCImach — APPROVED** [review]

File: src/components/Banner/Banners_.ts
Comment (Aliona-D) [inline src/components/Banner/Banners_.ts:1]: this file should be renamed to banners-styles.ts

File: src/components/Banner/Banners_.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  @media (max-width: 480px) {
+    border-bottom-right-radius: 30px;
+  }
+`;
+
+export const StyledPromotedBanner = styled(BaseBanner)``;
+
+export const StyledFlex = styled(Flex)`
```
Comment (Aliona-D) [inline src/components/Banner/Banners_.ts:53]: In such cases I avoid creating custom components but apply these styles directly in Flex component
<Flex direction={'column} justify={'start} align={'start'} width={'100%'} >

File: src/components/MarketCards/MarketCards_.ts
Comment (Aliona-D) [inline src/components/MarketCards/MarketCards_.ts:1]: this file should be renamed to market-cards-styles.ts

File: src/pages/Markets/MarketsFirstSection.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import { Box, Flex } from "@radix-ui/themes";
+import { OptionalLinkBanner } from "../../components/Banner/OptionalLinkBanner";
+import { PromotedBanner } from "../../components/Banner/PromotedBanner";
+
+export const FirstSection = () => {
+  return (
+    <Flex gap="3" style={{ marginLeft: "50px" }}>
```
Comment (Aliona-D) [inline src/pages/Markets/MarketsFirstSection.tsx:7]: here it can be simplified
<Flex gap="3" ml="50px">

File: src/pages/Markets/MarketsTable.tsx [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+import theme from "../../theme";
+import * as Select from "@radix-ui/react-select";
+// import OverlayLogo from "../../assets/images/overlay-logo-only-no-background.png";
+import { ChevronDownIcon, ChevronUpIcon } from "@radix-ui/react-icons";
+import { SelectItem } from "@radix-ui/react-select";
+import useSDK from "../../hooks/useSDK";
+import { useEffect, useState } from "react";
+import useMultichainContext from "../../providers/MultichainContextProvider/useMultichainContext";
+
+export type TransformedMarketData = {
```
Comment (Aliona-D) [inline src/pages/Markets/MarketsTable.tsx:13]: type TransformedMarketData should be exported from sdk.

File: src/pages/Markets/MarketsTable.tsx [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+export default function MarketsTable() {
+  const [marketsData, setMarketsData] = useState<TransformedMarketData[]>([]);
+  const { chainId: contextChainID } = useMultichainContext();
+  const sdk = useSDK();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const activeMarkets = await sdk.markets.transformMarketsData();
+        console.log("activeMarkets", activeMarkets);
```
Comment (Aliona-D) [inline src/pages/Markets/MarketsTable.tsx:34]: should be removed

File: src/pages/Markets/MarketsTable.tsx [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+        activeMarkets && setMarketsData(activeMarkets);
+      } catch (error) {
+        console.error("Error fetching markets:", error);
+      }
+    };
+
+    fetchData();
+  }, [contextChainID]);
+
+  console.log("marketsData", marketsData[1]);
```
Comment (Aliona-D) [inline src/pages/Markets/MarketsTable.tsx:45]: should be removed

File: src/theme/index.ts [hunk 8]
```diff
@@ -1,36 +1,38 @@
...
+    blue2: "#71CEFF",
+    blue3: "#12B4FF",
+    grey1: "#F2F2F2",
+    grey2: "#ECECEC",
+    grey3: "#A8A6A6",
+    grey4: "#252534",
+    grey5: "#373A44",
+    grey6: "#2E3343",
+    grey7: "#111111",
+    grey8: "#252534",
```
Comment (Aliona-D) [inline src/theme/index.ts:23]: grey8 = grey4, thus can be removed

File: src/theme/theme.ts
Comment (Aliona-D) [inline src/theme/theme.ts:1]: this file duplicates the previous one

**Pull request closed without merging**